tests/suites/verify.js

```diff
@@ -144,6 +144,18 @@ export function verifySuite({
           const credential = cloneTestVector(disclosed?.invalid?.valuePrefix);
           await verificationFail({credential, verifier});
         });
+        it('If the result is not an array of five, six, or seven elements ' +
+          '— a byte array, a map of integers to integers, two arrays of ' +
+          'integers, and one or two byte arrays; an error MUST be raised ' +
+          'and SHOULD convey an error type of PROOF_VERIFICATION_ERROR.',
```

Comment:
This enumeration adds up to 5 or 6 elements. What's the missing (possible) element? Also, the enumeration should be separated from the surrounding text by surrounding em-dashes, not introduced by em-dash and ended by semicolon.
```suggestion
        it('If the result is not an array of five, six, or seven elements ' +
          '— a byte array, a map of integers to integers, two arrays of ' +
          'integers, and one or two byte arrays — an error MUST be raised ' +
          'and SHOULD convey an error type of PROOF_VERIFICATION_ERROR.',
```